e-shop-website-with-ilb-ase/azuredeploy.parameters.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "existingASEName": {
+      "value": "GET-PREREQ-existingASEName"
+    },
+    "existingASELocation": {
+      "value": "GET-PREREQ-existingASELocation"
+    },
+    "existingASEDNSSuffix": {
+      "value": "GET-PREREQ-existingASEDNSSuffix"
+    },
+    "dnsSuffix": {
```

Comment:
remove this and just use a defaultValue in the template

Reply:
done